src/tests/simple_tests/testdata/pubdata_refunds/entry.asm

```diff
@@ -0,0 +1,65 @@
+    .text
+    .file "Test_26"
+    .rodata.cst32
+    .p2align    5
+    .text
+    .globl __entry
+__entry:
+.main:
+    ; allocate 20k gas for 2 writes
+    ; each write takes about 5k gas
+    ; add 20000, r0, r4
+    ; 200k
+    add 500000, r0, r4
+    
+    ; perform write via near call
+    near_call r4, @inner_s_write, @.panic
+
+    ret.ok r0
+inner_s_write:
+    ; prepare a 32-bit mask (0xffff..)
+    add 1, r0, r10
+    shl.s 32, r10, r10
+    sub.s 1, r10, r10
+    
+    ; check pubdata counter (should be equal to 0)
+    context.meta r7
+    and r10, r7, r7
+    sub! 0, r7, r0
+    jump.ne @.panic
+
+    ; we'll be writing 13 at slot 25 with a warm refund of 5400
```

Comment:
I think that we can implement a little bit better approach. Our tracer can receive "commands" that are written to `r0` (add X, r0, r0). So we can add new type of command (and in asm preprocessing too)

For example, we can create flow like:
1. command: "S:{0,1}" - tracer executes command "reset to cold storage access" or moves to state ExpectRefundValue.
2. If tracer in "ExpectRefundValue" it will interpet next "add X, r0, r0" such that X is refund value

Reply:
oh good idea! will try that out

Reply:
It will be very close to "print register" implementation here:
https://github.com/matter-labs/era-zkevm_test_harness/blob/d16297e5158968ae2cfc539f297df89ce1498343/src/tests/utils/testing_tracer.rs#L165

And also you can add new kind of directive here (And implement it further):
https://github.com/matter-labs/era-zkevm_test_harness/blob/d16297e5158968ae2cfc539f297df89ce1498343/src/tests/utils/preprocess_asm.rs#L24

For example we can do something like `set_storage_warm(refund)` and `set_storage_cold()`

Reply:
that's a very nice idea.